packages/calypso-e2e/src/lib/pages/cart-checkout-page.ts

```diff
@@ -1,11 +1,42 @@
-import { Page } from 'playwright';
+import { Frame, Page } from 'playwright';
+import { DataHelper } from '../..';
+import { getTargetDeviceName } from '../../browser-helper';
+import type { PaymentDetails } from '../../data-helper';
+import type { TargetDevice } from '../../types';
 
 const selectors = {
-	modalContinueButton: 'button:text("Continue")',
+	// Modal
+	modalContinueButton: '.checkout-modal__content button:text("Continue")',
+
+	// Banner
+	dismissBanner: `button[aria-label="Dismiss"]`,
+
+	// Cart item
 	cartItem: ( itemName: string ) =>
 		`[data-testid="review-order-step--visible"] .checkout-line-item >> text=${ itemName.trim() }`,
 	removeCartItemButton: ( itemName: string ) =>
 		`[data-testid="review-order-step--visible"] button[aria-label*="Remove ${ itemName.trim() } from cart"]`,
+
+	// Tax information
+	countryCode: `select[aria-labelledby="country-selector-label"]`,
+	postalCode: `input[id="contact-postal-code"]`,
+	submitBillingInformationButton: `button[aria-label="Continue with the entered contact details"]`,
+
+	// Payment field
+	cardholderName: `input[id="cardholder-name"]`,
+	cardNumberFrame: 'iframe[title="Secure card number input frame"]',
+	cardNumberInput: 'input[data-elements-stable-field-name="cardNumber"]',
```

Comment:
Within the card number iframe there are multiple hidden input tags, so we must explicitly look for the correct field.